core/frontend/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": false,
```

Comment:
This should be true, as I said in my previous comment, we should use the strict mode to avoid unnecessary pain and problems in the future as developers.